integration_test/metadata/integration_metadata.go

```diff
@@ -49,6 +49,9 @@ type ExpectedMetric struct {
 	Representative bool `yaml:"representative,omitempty" validate:"excluded_with=Optional,excluded_with=Platform"`
 	// Exclusive metric to a particular kind of platform.
 	Platform string `yaml:"platform,omitempty" validate:"excluded_with=Representative,omitempty,oneof=linux windows"`
+	// A list of platforms that this metric is not available on.
+	// Examples: centos-7,debian-10. Not valid are linux,windows.
+	UnavailableOn []string `yaml:"unavailable_on,omitempty" validate:"excluded_with=Representative"`
```

Comment:
I wonder if this information also needs to be propagated into the generated documentation… @sophieyfang WDYT?

Reply:
sophie and i discussed this a bit, and we agreed that since we don't surface "optional" for log fields or metrics in the docs, we don't need to surface "unavailable_on" either.

If we decided to surface that information later, we could do it by saying something like "this metric/log field may be unavailable on some platforms" if optional or unavailable_on are specified.

regarding sophie's suggestion to use "optional' instead of 'unavailable_on" for log fields, we haven't decided yet. I'm OK with that approach but I think unavailable_on is going to produce slightly better test coverage. The difference is probably minimal though.